Makefile

```diff
@@ -197,15 +197,6 @@ pao-functests-only:
 	hack/show-cluster-version.sh
 	hack/run-functests.sh
 
-.PHONY: pao-functests-updating-profile
```

Comment:
is this updated with latest ?   where did the targets go ?

Reply:
Ha.  Great catch!  No idea what happened there :/  Fixed, thank you!